scripts/http-fetch.nse

```diff
@@ -47,18 +47,17 @@ The script supports three different use cases :
 -- @output
 -- | http-fetch:
 -- |   Successfully Downloaded:
--- |     http://nmap.org:80/ as /tmp/mirror/index.html
--- |     http://nmap.org/shared/css/insecdb.css as /tmp/mirror/shared/css/insecdb.css
--- |     http://nmap.org/movies/ as /tmp/mirror/movies/index.html
--- |     http://nmap.org/book/man.html as /tmp/mirror/book/man.html
---
+-- |     http://scanme.nmap.org:80/ as /tmp/mirror/45.33.32.156/80/index.html
+-- |     http://scanme.nmap.org/ as /tmp/mirror/45.33.32.156/80/index.html
```

Comment:
This line is a logical duplicate of the previous line. Should they both be here?

Reply:
Oh, sorry I didn't notice that. Maybe example scan of `nmap.org` would be better than `scanme.nmap.org`.
But if they are the same, why is `http-fetch` reporting them multiple times. This can be an issue in itself, maybe something to do with the `fetch_recursively` function [here](https://github.com/nmap/nmap/blob/master/scripts/http-fetch.nse#L129).

This is the output on scanning `scanme.nmap.org`
```
| http-fetch: 
|   Success Downloaded: 
|     http://scanme.nmap.org:80/ as /tmp/mirror/45.33.32.156/80/index.html
|     http://scanme.nmap.org/shared/css/insecdb.css as /tmp/mirror/45.33.32.156/80/shared/css/insecdb.css
|_    http://scanme.nmap.org/ as /tmp/mirror/45.33.32.156/80/index.html
```
I'll work on this issue and try to fix the above repetition meanwhile @nnposter